src/components/views/DownloadModModal.vue

```diff
@@ -184,47 +184,35 @@ import ProfileModList from '../../r2mm/mods/ProfileModList';
             }, 1);
         }
 
-        static async installModAfterDownload(profile: Profile, mod: ThunderstoreMod, version: ThunderstoreVersion): Promise<R2Error | void> {
-            return new Promise(async (resolve, reject) => {
+        static async installModAfterDownload(profile: Profile, mod: ThunderstoreMod, version: ThunderstoreVersion): Promise<void> {
```

Comment:
I was going to write that we need to change the call site to handle errors no longer being returned but seems they haven't been handler previously? Now they are though. I _guess_ that's a good thing?